ClassicPress-plugins/classic-commerce
# Update colors of CC images to aqua

Replaces the following images:

`assets/images/banner-772x250.jpg`
`assets/images/classic-commerce-logo.png`

Closes #254

### How to test the changes in this Pull Request:

1. Replace the above images locally with those in this PR
2. Check setup screen and plugin details page
